src/main/java/org/dormitory/autobotsoccub/command/querycommand/UnregisterCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.dormitory.autobotsoccub.command.querycommand;
+
+import lombok.AllArgsConstructor;
+import org.dormitory.autobotsoccub.command.InlineQueryCommand;
+import org.dormitory.autobotsoccub.command.keyboard.Button;
+import org.dormitory.autobotsoccub.command.keyboard.KeyboardFactory;
+import org.dormitory.autobotsoccub.command.result.CommandResult;
+import org.dormitory.autobotsoccub.user.UserPool;
+import org.telegram.telegrambots.api.objects.Update;
+import org.telegram.telegrambots.api.objects.User;
+
+import static org.dormitory.autobotsoccub.command.keyboard.Button.*;
+
+@AllArgsConstructor
+public class UnregisterCommand implements InlineQueryCommand {
+
+    private KeyboardFactory keyboardFactory;
+
+    private UserPool userPool;
+
+    @Override
+    public Button getCommandButton() {
+        return Button.UNREGISTER;
+    }
+
+    @Override
+    public CommandResult execute(Update update) {
+        User currentUser = update.getCallbackQuery().getFrom();
+        if (userPool.containsUser(currentUser.getId())) {
+            userPool.removeUser(currentUser);
+            return CommandResult.builder()
+                    .replyMessage(currentUser.getFirstName() + ", your registration for the game is canceled! But you can change your mind...")
```

Comment:
[future] the same as for register command; we gonna decrease unnecessary level of verbosity

Reply:
The same here)